src/earn/EarnCollectScreen.test.tsx

```diff
@@ -97,6 +97,10 @@ describe('EarnCollectScreen', () => {
         (gateName: StatsigFeatureGates) => gateName === StatsigFeatureGates.SHOW_POSITIONS
       )
     jest.mocked(isGasSubsidizedForNetwork).mockReturnValue(false)
+    jest.mocked(getEarnPositionBalanceValues).mockReturnValue({
```

Comment:
can we just require actual instead of mocking this, there are a few ways you can do this.

Reply:
fixed in [43c1e4a](https://github.com/valora-inc/wallet/pull/6155/commits/43c1e4ad3cd45253982ea92d977b4ec78346c0f8)